Objects/longobject.c

```diff
@@ -410,6 +414,30 @@ PyLong_FromUnsignedLong(unsigned long ival)
     return (PyObject *)v;
 }
 
+/* Same as above, but for unsigned long. */
```

Comment:
Might make sense to simply remove these comments, because it's quite obvious that they do the same thing as `long_from_uint()`.

Reply:
Non-existing comments never lie and don't require updates :slightly_smiling_face: .

Reply:
@sir-sigurd:
> Non-existing comments never lie and don't require updates :slightly_smiling_face:.

Between the choice of keeping the comment as is vs removing it, I would support removing it more. But, I think it would be useful to readers to mention @gnprice's suggestion (removing the "C" part as well):

>Create a new int object from an unsigned long

I think it's fairly common for developers to look at the implementation for a single function in isolation rather than reading the entire file at once, so it's not always optimal to rely on comments in earlier sections for context. To some degree, each function should aim to explain itself.